client/components/Header.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+export default class Header extends React.Component {
+    render(){
+        return (
+            <header>
+                <nav className="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+                    <a className="navbar-brand" href="#">Dashboard</a>
+                    <button className="navbar-toggler d-lg-none" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
+                    <span className="navbar-toggler-icon"></span>
+                    </button>
+                    <div className="collapse navbar-collapse" id="navbarsExampleDefault">
+                    <ul className="navbar-nav mr-auto">
+                        <li className="nav-item active">
+                        <a className="nav-link" href="#">Home <span className="sr-only">(current)</span></a>
+                        </li>
+                        <li className="nav-item">
+                        <a className="nav-link" href="#">Settings</a>
+                        </li>
+                        <li className="nav-item">
+                        <a className="nav-link" href="#">Profile</a>
+                        </li>
+                        <li className="nav-item">
```

Comment:
See this nav-item li. It can also be converted into a new component and can be reused